### DISCUSSION TITLE
Language selection in the settings doesn't update after choosing a different language in the setup wizard

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#21743] [Q&A] [answered]
i feel like i've seen this issue before but i couldn't find anything when i was looking for it

https://user-images.githubusercontent.com/43358824/208829768-e0a74c5e-55b0-4b90-ab75-27935388ff30.mp4

testing on commit d81a724dd7141818ecd76bf057147f98dc2ccda1

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21744
